src/collections/_documentation/development/sdk-dev/unified-api.md

```diff
@@ -86,7 +94,7 @@ Additionally it also setups all default integrations.
 
 - `capture_message(message, level)`: Reports a message. The level can be optional in language with default parameters in which case it should default to `info`.
 
-- `add_breadcrumb(crumb)`: Adds a new breadcrumb to the scope. If the total number of breadcrumbs exceeds the `max_breadcrumbs` setting, the oldest breadcrumb should be removed in turn. This works like the Hub api with regards to what `crumb` can be.
+- `add_breadcrumb(crumb)`: Adds a new breadcrumb to the scope. If the total number of breadcrumbs exceeds the `max_breadcrumbs` setting, the oldest breadcrumb should be removed in turn. This works like the Hub api with regards to what `crumb` can be. The breadcrumb is ignored if the SDK is disabled.
```

Comment:
> The breadcrumb is ignored if the SDK is disabled.
Do we really want to define it that strictly?
Why wouldn't we want that even if disabled it will keep the breadcrumbs?

Reply:
While I don’t like making public API based on implementation details in general, here, if we don’t have a client, we don’t have a `max_breadcrumbs` setting. So what should we do in that case? Maybe define a global limit somewhere?

Also, both this and `configure_scope` live on the scope. So we should skip both, to at least be consistent.

Reply:
`max_breadcrumbs` lives on SentryOptions though.